ansible_collections/arista/avd/examples/cv-pathfinder/group_vars/WAN/management.yml

```diff
@@ -0,0 +1,108 @@
+---
+# WAN Management settings
+mgmt_gateway: 192.168.17.1
+# mgmt_gateway: 10.90.226.1
+
+# NTP
+ntp_settings:
+  servers:
+    - name: 0.pool.ntp.org
+
+# CloudVision/TerminAttr
+# **edit the cvp_instance_ips to match your CVaaS region**
+cvp_instance_ips: [apiserver.cv-staging.corp.arista.io]
+terminattr_smashexcludes: "ale,flexCounter,hardware,kni,pulse,strata"
+terminattr_ingestexclude: "/Sysdb/cell/1/agent,/Sysdb/cell/2/agent"
+
+# For each local user in this example, password == username
+local_users:
+  - name: arista
+    privilege: 15
+    role: network-admin
+    sha512_password: $6$Enl0WfE32FthwyiJ$yTyGaEJ2uPKLU.F7314YtB7J1jrzrMi7ogXIRTEHQfLdLgKWWmr1UvNlZLN6AyuxET7G5aH3AI9OYRzxVTkB1.
+  - name: cvpadmin
+    privilege: 15
+    role: network-admin
+    sha512_password: $6$a7LdQWHxWzYHpvVt$n62q.1mbm4kzQ5oBr0lhXCE9ntnTn.SNa16DovZHahFQLH.iPcPMZa5JUSFtncrDW4EDQ3oSWgP8G0S4FtOFx1
+
+# Internet ACL
+ipv4_acls:
+  - name: ACL-PF-INTERNET-IN
+    entries:
+      - sequence: 1
+        remark: "Not for PRODUCTION: This ACL is built this way because the lab has an out-of-band interface"
+      - sequence: 10
+        # IPSEC taffic from anywhere
+        action: permit
+        protocol: udp
+        source: any
+        destination: interface_ip
+        destination_ports_match: eq
+        destination_ports: [isakmp, non500-isakmp]
+      - sequence: 20
+        # STUN taffic from anywhere
+        action: permit
+        protocol: udp
+        source: any
+        destination: interface_ip
+        destination_ports_match: eq
+        destination_ports: [3478]
+      - sequence: 30
+        # Troubleshooting
+        action: permit
+        protocol: icmp
+        source: any
+        destination: interface_ip
+      - action: deny
+        protocol: ip
+        source: any
+        destination: any
+  - name: ACL-INTERNET-IN
```

Comment:
I don't see that we allow return traffic for STUN.

Reply:
@ClausHolbechArista Would that be handled here https://github.com/aristanetworks/avd/pull/4453/files#diff-bf22f0d841e82a7b205f1df64958756fbf25da1d8b690e250a30873dbf9d6c92R35

Reply:
Disregard that comment, I see what you mean :)